### ISSUE TITLE
Unknown scene handler

### ISSUE THREAD

**Issue by Implausiblyfun** [oakmound/oak#139]
The specifics of this issue will depend on whether it should be implemented for oak 3 or not asscene management will likely change in 3.

Today oak throws a panic if it attempts to be sent to an unknown scene. While this makes sense most of the time due to the fact that scenes should be all set up ahead of time it seems like there could be a possibility that an implementer runs a little bit fast and loose with this concept.
Long story short I dont like panics and propose an optional setting that will control logic when attempting to access an unknown scene.
Easy example would be secret panic recovery scene or some such that routes back to first scene or some such.
